src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
+  "DASHBOARD.VIEW_ALL": "View all {servicesCount} Services",
```

Comment:
@MatApple the lib also support plural but in this case it's Services one or many based on the UI and the code we had in place so I only "replicated" using the localization.

That being said though I have seen two ways to do that so far. One is to use the
[string formatting](https://github.com/yahoo/react-intl/wiki/Components#string-formatting-components) and the other is by composing `FormattedMessage` component with `FormattedPlural` where we would add another key/value in value object.